src/outputs/terraform/aws-ec2/cndi_aws_volume_attachment.tf.json.ts

```diff
@@ -0,0 +1,24 @@
+import { getPrettyJSONString, getTFResource } from "src/utils.ts";
+import { AWSEC2NodeItemSpec } from "src/types.ts";
+
+export default function getAWSComputeInstanceTFJSON(
+  node: AWSEC2NodeItemSpec,
+): string {
+  const { name } = node;
+  const instance_id = `\${aws_instance.cndi_aws_instance_${name}.id}`; // ID of the Instance to attach to
+  const volume_id = `$\{aws_ebs_volume.cndi_aws_ebs_volume_${name}.id}`; // ID of the Volume to be attached
+  const device_name = "/dev/sdf"; // The device name to expose to the instance
```

Comment:
this was previously `/dev/sda1`, did we change this intentionally?

Reply:
That name is reserved for the root volume that holds the image and OS

Reply:
So we can't use it with volume attachments

Reply:
root volumes are where everything is running by default as I understand it, I think by using a non-root device we are attaching a second drive instead of modifying the original disk

Reply:
![image](https://github.com/polyseam/cndi/assets/77031428/78469efd-0928-413c-a8a4-959e4eb25b94)

Reply:
do we need to tell software which disk to write to?

Reply:
I don't think so, because gcp basically works the same way